okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ConcurrencyLimiters.java

```diff
@@ -101,12 +101,12 @@ private ConcurrencyLimiter newLimiter(String limiterKey) {
         return new ConcurrencyLimiter(limiterKey, limiter);
     }
 
-    private static String limiterKey(Request request) {
+    private String limiterKey(Request request) {
         String pathTemplate = request.header(OkhttpTraceInterceptor.PATH_TEMPLATE_HEADER);
         if (pathTemplate == null) {
             return FALLBACK;
         } else {
-            return request.method() + " " + pathTemplate;
+            return request.url().host() + " " + request.method() + " " + pathTemplate;
```

Comment:
I think it’s time to key on a proper object

Reply:
heh, yes

Reply:
done